plugins/kubernetes/app/models/kubernetes/resource_status.rb

```diff
@@ -40,6 +40,16 @@ def check
         else
           @details = "Waiting (#{@pod.phase}, #{@pod.reason})"
         end
+      elsif kind == "HorizontalPodAutoscaler"
+        hpa = Kubernetes::Api::HorizontalPodAutoscaler.new
+        failures = hpa.events_indicating_failure(events(type: "Warning"))
+        if failures && !failures.empty?
```

Comment:
can use `if failures = hpa.events_indicating_failure(events(type: "Warning")).presence` to get rid of the empty/nil check

Reply:
also failures will never be nil so `if failures.any?` would do too